quodlibet/po/fr.po

```diff
@@ -776,7 +776,7 @@ msgstr "Impossible de copier le titre"
 
 #: ../quodlibet/browsers/media.py:514
 msgid "There is not enough free space for this song."
-msgstr "Espace insuffisant pour ce titre"
+msgstr "Espace libre insuffisant pour ce titre"
```

Comment:
Maybe we could add back the final dot, that can be found in the English version.